eip/EIPS/eip-4337.md

```diff
@@ -309,6 +309,7 @@ While simulating `userOp` validation, the client should make sure that:
    4. cannot call EntryPoint's methods, except `depositFor` (to avoid recursion)
 5. `EXTCODEHASH` of every address accessed (by any opcode) does not change between first and second simulations of the op.
 6. `EXTCODEHASH`, `EXTCODELENGTH`, `EXTCODECOPY` may not access address with no code.
+The exception is accessing the `sender` address from the `factory` contract as it is guaranteed to deploy the `sender`.
```

Comment:
`may not access address with no code (except the sender address)`
The exception for sending address is because it can only be accessed from factory ..